packages/dd-trace/src/appsec/iast/taint-tracking/rewriter.js

```diff
@@ -10,12 +10,51 @@ const { getRewriteFunction } = require('./rewriter-telemetry')
 
 let rewriter
 let getPrepareStackTrace
+
+let getRewriterOriginalPathAndLineFromSourceMap = function (path, line, column) {
+  return { path, line, column }
+}
+
+function isEnableSourceMapsFlagPresent () {
```

Comment:
Source Map support could be also enabled by `NODE_OPTIONS` environment variable. With the current approach we are missing the enablement via environment variable.

[NODE_OPTIONS documentation](https://nodejs.org/api/cli.html#node_optionsoptions)

Reply:
good point!